content/en/security/cloud_siem/log_detection_rules.md

```diff
@@ -326,6 +326,10 @@ Use the **Tag resulting signals** dropdown menu to add tags to your signals. For
 
 **Note**: the tag `security` is special. This tag is used to classify the security signal. The recommended options are: `attack`, `threat-intel`, `compliance`, `anomaly`, and `data-leak`.
 
+## Suppression rules
```

Comment:
LGTM - one small nit - can we move the section "Exclude benign activity with suppression queries" to right above the "suppression rules" section?